app/api/v1/endpoints/memory/endpoints.py

```
@@ -3488,11 +3699,24 @@ async def update_memory(
             else existing.get("metadata", {})
         )
 
+        reviewer_id: UUID | None = None
+        if admin_user.sub:
+            try:
+                parsed_reviewer = UUID(admin_user.sub)
+                if parsed_reviewer.int != 0:
+                    reviewer_id = parsed_reviewer
+            except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)